authors/apps/authentication/models.py

```diff
@@ -116,5 +118,35 @@ def get_short_name(self):
         the user's real name, we return their username instead.
         """
         return self.username
-
-
+    @property
+    def generate_password_reset_token(self):
+        # methods that generates an email activation token
+        if self.email is not None:
+            payload = {
+                'id':self.pk,
+                'email':self.email,
+                'username':self.username,
+                'exp': datetime.utcnow() + timedelta(minutes=20)
+            }
+            token = jwt.encode(payload,
+                               os.environ.get('SECRET_KEY',
```

Comment:
Get the secret key from settings to avoid duplication.

Reply:
worked on that and also decide to use the token generation method created by Emma since there was no point implementing the same method